seed/static/seed/js/directives/sdUploader.js

```diff
@@ -161,146 +161,6 @@ var makeFileSystemUploader = function (scope, element, allowed_extensions) {
   return uploader;
 };
 
-var makeBuildingSyncUploader = function (scope, element, allowed_extensions) {
```

Comment:
Nice! We remove this because BuildingSync files now use the "normal" upload flow right?

Reply:
That's correct 👍